cogs/utils/context.py

```diff
@@ -140,14 +143,14 @@ def check(payload):
                 return True
             for i in range(additional_options):
                 if codepoint == f'{i + 1}\N{combining enclosing keycap}':
-                    confirm = i
+                    confirm = i + 1
                     return True
 
             return False
 
         if not additional_options:
             await msg.add_reaction('\N{WHITE HEAVY CHECK MARK}')
-        await msg.add_reaction('\N{CROSS MARK}')
+            await msg.add_reaction('\N{CROSS MARK}')
```

Comment:
I actually left the cross there for autoclaim in case none of the options were right you hit "x" to skip to next one

Reply:
That actually makes a lot of sense. :)